docs/UserGuide.md

```diff
@@ -239,6 +239,49 @@ Jobby data are saved automatically as a JSON file `[JAR file location]/data/jobb
 If your changes to the data file makes its format invalid, Jobby will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.
 </div>
 
+### Applying to an organization: `Apply`
+
+Format: `add index/ id <additional parameters and values...>`
+
+
+App a contact to the address book of the given class type: Organization or Recruiter.
+
+Applying to a Organization by indicating it with the organization's index in the address book or the organization's unique id.
+
+This can be done by supplying the organization's `index` or `id` as the preamble.
+
+Details specifically will be explained the next sections.
+
+
+#### Applying to an Organization: `add org index/org id`
+
```

Comment:
ps: command formats are incorrect; there are a few typos around here; note that `INDEX` and `ORG` should be capitalized by convention and put together with no space in `INDEX/ORG`